region/lowernorfair/east/Wasteland.json

```diff
@@ -641,6 +669,57 @@
       "clearsObstacles": ["A", "B"],
       "flashSuitChecked": true
     },
+    {
+      "link": [4, 4],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "requires": [
+        {"obstaclesCleared": ["R-Mode"]},
+        {"obstaclesNotCleared": ["C"]},
+        "h_heatProof",
+        {"or": [
+          {"and": [
+            "h_heatedCrystalFlash",
+            {"obstaclesCleared": ["D"]}
+          ]},
+          {"and": [
+            {"obstaclesNotCleared": ["D"]},
+            "h_RModeCanRefillReserves",
+            {"or": [
+              {"and": [
+                "canBePatient",
+                {"partialRefill": {"type": "ReserveEnergy", "limit": 20}},
+                {"enemyKill": {"enemies": [["Dessgeega"], ["Dessgeega", "Dessgeega", "Dessgeega"]], "excludedWeapons": ["Missile"]}}
+              ]},
+              {"and": [
+                {"or": [
+                  {"partialRefill": {"type": "ReserveEnergy", "limit": 5}},
+                  {"and": [
+                    "canBeVeryPatient",
+                    {"partialRefill": {"type": "ReserveEnergy", "limit": 20}}
+                  ]}
+                ]},
+                {"enemyKill": {"enemies": [["Dessgeega"], ["Dessgeega", "Dessgeega", "Dessgeega"]]}}
```

Comment:
Charge and Morph Bombs probably need patience.
Pseudo screw is out.

Reply:
Hopefully excluding Bombs/Charge from the "not patient" route works. I think this only excludes Charge Power Beam so if I need to come back and get Charge + (some mix of Ice/Wave/Spazer) I could add those.

Plasma (with or without Charge) definitely should not require patience.